app/graphql/inputs/user/login.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Inputs::User
+  class Login < Inputs::BaseInputObject
+    description "Credentials necessary for user login"
+    # the name is usually inferred by class name but can be overwritten
+    graphql_name "UserLoginInput"
+
+    argument :login, String, required: true
+    argument :password, String, required: true
+    argument :remember_me, Boolean, required: false
```

Comment:
Do you want/does this need a default?

Reply:
Still researching this. I believe it defaults to null in any case... is there a more desirable default? `required: nullable` is a possibility. 

The [argument docs](https://graphql-ruby.org/fields/arguments.html) lay out the options, but i haven't found a login input example out there with default values yet.